### PR TITLE
Fixes #71 | Add Dockerfile

### LINKED ISSUE #71: Dockerfile
Hi Guys, I was using the original ciscocloud/marathon-consul in a docker image (for coreos) and used the Dockerfile supplied with the project. I have created a Dockerfile by pulling down the distro. I would prefer to build from source, I am a bit lost how to create a Dockerfile for your project as I have no real experience with golang... Can you anyone help please?

======================

Ok, managed to hack the ciscocloud/marathon-consul file, very simple just used the Makefile and removed the test tag from the build target.

The launch.sh is again from ciscocloud/marathon-consul 

===== DOCKERFILE =====
FROM gliderlabs/alpine:3.2
MAINTAINER Brian Hicks <brian@brianthicks.com>

RUN apk add --update ca-certificates bash
COPY launch.sh /launch.sh

COPY . /go/src/github.com/allegro/marathon-consul
RUN apk add make go git mercurial \
  && cd /go/src/github.com/allegro/marathon-consul \
  && export GOPATH=/go \
  && make build \
  && mv /go/src/github.com/allegro/marathon-consul/bin/marathon-consul /bin/marathon-consul \
  && rm -rf /go \
  && apk del --purge go git mercurial

ENTRYPOINT ["/launch.sh"]

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM scratch
+MAINTAINER Tomasz Janiszewski <janiszt@gmail.com>
```
Comment (xor-xor) [inline Dockerfile]: Are you sure about this..? ;)

File: README.md [hunk 2]
```diff
@@ -77,6 +77,22 @@ sudo apt-get -qy install marathon-consul
...
+
+To build docker image run
+```bash
+make docker
+```
+Then you can run it with
+```bash
+docker run -d -P allegro/marathon-consul [options]
+```
+If you want to use SSL you will need to expose cert store to Docker. Following line is only example, your cert store
```
Comment (xor-xor) [inline README.md]: s/Following/The following/

File: README.md [hunk 3]
```diff
@@ -77,6 +77,22 @@ sudo apt-get -qy install marathon-consul
...
+might be different depends on your system.
```
Comment (xor-xor) [inline README.md]: s/depends/depending/

**Comment by xor-xor** [comment]
LGTM

File: README.md (hunk 2 above)
Comment (janisz) [inline README.md]: Fixed

File: Dockerfile (hunk 1 above)
Comment (janisz) [inline Dockerfile]: Fixed

Comment (janisz) [inline Dockerfile]: @adamdubiel @vi4m What should be here? See [other Allegro repos](https://github.com/search?l=dockerfile&q=MAINTAINER++user%3Aallegro&ref=searchresults&type=Code&utf8=%E2%9C%93)

Comment (dankraw) [inline Dockerfile]:
```
MAINTAINER Allegro
```
should be fine.

Comment (janisz) [inline Dockerfile]: Fixed

**Comment by dankraw** [comment]
👍

**Comment by janisz** [comment]
@jogster @tsaridas Is this working four you?

**Pull request merged**